addons/atragmx/functions/fnc_update_range_card.sqf

```diff
@@ -43,7 +43,7 @@ _speedOfSound = GVAR(temperature) call EFUNC(weather,calculateSpeedOfSound);
     _TOF =  _x select 4;
     _velocity = _x select 5;
     _kineticEnergy = _x select 6;
```

Comment:
I'll make another PR to find possible usage of params later, this one is mostly about whitespace.